src/Compilers/Extension/source.extension.vsixmanifest

```diff
@@ -17,6 +17,6 @@
     <Asset Type="Microsoft.VisualStudio.VsPackage" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" d:Source="Project" d:ProjectName="%CurrentProject%" />
   </Assets>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,17.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,18.0)" DisplayName="Visual Studio core editor" />
```

Comment:
Why aren't we moving to the version range `[17.0,18.0)`? At this point we don't support deploying to pre-Dev17 hence not sure why we are keeping the minimum version at 15.

Reply:
There are a bunch of version anomalies in these files. They seem to exist primarily to support integration testing; it's not clear how much effort needs to go into making them match what would go on marketplace, but if you want specific consistency check we can implement one.